tests/basic.test.tsx

```diff
@@ -1,6 +1,7 @@
 import {
   Component as ClassComponent,
   ReactNode,
+  version as reactVersion,
```

Comment:
I didn't know this. I should use this in [jotai](https://github.com/pmndrs/jotai) tests.

Reply:
just did so: https://github.com/pmndrs/jotai/pull/1287

Reply:
I like how you use regexp, gonna use that here too.